Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Marking -t option in lxc-create manpage as required (Closes: #768778),
 see https://github.com/lxc/lxc/issues/355.

diff -Naurp lxc.orig/doc/lxc-create.sgml.in lxc/doc/lxc-create.sgml.in
--- lxc.orig/doc/lxc-create.sgml.in
+++ lxc/doc/lxc-create.sgml.in
@@ -51,7 +51,7 @@ Foundation, Inc., 51 Franklin Street, Fi
       <command>lxc-create</command>
       <arg choice="req">-n <replaceable>name</replaceable></arg>
       <arg choice="opt">-f <replaceable>config_file</replaceable></arg>
-      <arg choice="opt">-t <replaceable>template</replaceable></arg>
+      <arg choice="req">-t <replaceable>template</replaceable></arg>
       <arg choice="opt">-B <replaceable>backingstore</replaceable></arg>
       <arg choice="opt">-- <replaceable>template-options</replaceable></arg>
     </cmdsynopsis>
